# Analyzing US Museum Websites with Web Archives

This notebook was created during the [Archives Unleashed] event held in Washington DC on June 14-15.

## Background

[IMLS] maintains the [Museums Universal Data File] which is a census of museums in the United States. Conveniently the database is available for download as a [dataset] in CSV format. The [ArchiveIt] folks at Internet Archive pulled the website URLs from the CSV and then used them to create a seed list for a (big) web crawl. The 10 gigabytes of [CDX files] for those crawls were made available as a dataset for use during Archives Unleashed.

With your archivist hat on (if you have one) you can think of [CDX] files as a *finding aid* for the Web crawl data, or WARC data files. They document what URLs were requested, what the [response code] was (404 Not Found, 500 Internal Server Error, 200 OK, etc), the size of the response, the media type of the response (text/html, image/jpeg, etc), as well as other metadata to aid in seeking within the CDX file and in the corresponding WARC file to get the actual content.

## Processing

We thought that perhaps it might be interesting to augment the IMLS data file with information from the crawl, such as the total number of pages on the museums website, the total size of resources hosted, the number of missing resources, etc. In addition we thought it could be interesting to look at the data through the lens of the museum's income.

To be able to augment the IMLS table with data from the CDX file I chose to read the CDX file and count attributes of the request (status code, mimetype, size, etc) using [Redis]. Unfortunately we found out during the processing that there is a great deal of duplicate URLs in the crawl: a given URL may have been crawled many times. So we also needed to only tally the first occurrence of the resource, and ignore the rest, so that our results don't favor resources that were requested many times by the crawler. You can see the resulting program here in [index.py].

Once the data has been processed into Redis you can then run [augment.py] which will read in the existing CSV file, and write out a new CSV with the additional columns: 

* pages: number of *unique* pages requested
* size: total size in bytes received from a domain
* 404: number of Not Found errors at a domain
* 500: number of Internal Server Errors at a domain
* robots: 

More columns could be added, such as number of images (jpeg, tiff, etc) if desirable. This notebook uses the [augmented CSV dataset] to examine a few things.

## Caveats:

* These results are preliminary, index.py is still running after 10 hours! It appears to be 75% done, so these results are impartial. Maybe I should have tried [ArchiveSpark] or something.  Sigh. I'll update the notebook with the latest CSV when it completes.
* Any Website URLs that were at a path within a larger site were ignored in this analysis. The reason being that it is not clear where one website begins and another ends unless there is a naked domain.
* Naked domains are only a limited view of a particular organizations web space. It could be that they have a lot of content at another domain.
* The web crawl ran for 3 months, but still hasn't completed. According to [Jefferson] it apparently is stuck in some NFL website at the moment.


[augmented CSV dataset]: imls-2015-augmented.csv
[Jefferson]: https://twitter.com/jefferson_bail
[IMLS]: http://www.imls.gov
[Redis]: http://redis.io
[ArchiveIt]: http://archiveit.org
[Archives Unleashed]: http://archivesunleashed.com
[Museums Universal Data File]: https://www.imls.gov/research-evaluation/data-collection/museum-universe-data-file
[dataset]: https://data.imls.gov/Museum/Museum-Universe-Data-File-FY-2015-Q1/bqh6-bapa
[CDX files]: http://qa-server.us.archive.org/vinay-misc/imls-cdx/cdx-manifest.txt
[WARC]: https://en.wikipedia.org/wiki/Web_ARChive
[CDX]: https://archive.org/web/researcher/cdx_file_format.php
[response code]: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
[index.py]: index.py

## Analysis

Let's get started by importing a few things. We're going to use pandas, and maybe view some things with matplotlib.

In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

%matplotlib notebook


In [2]:
imls = pd.read_csv("imls-2015-augmented.csv")

So here's what our table looks like:

In [3]:
imls.head()

Name                     URL     Income  Pages  \
0                      ARTSREVIVE          artsrevive.com   138290.0      0   
1              REV BIRMINGHAM INC   www.revbirmingham.org  1695242.0      0   
2    BELLINGRATH GARDENS AND HOME     www.bellingrath.org  3422595.0   2101   
3  PIONEER MUSEUM OF ALABAMA ASSN  www.pioneer-museum.org   103113.0    109   
4     ALABAMA SPORTS HALL OF FAME               ashof.org   552152.0   2230   

        Size  404  500 Robots  
0          0    0    0  False  
1          0    0    0  False  
2  768279385   10    0   True  
3   69725620    2    0   True  
4   33108465   24    0  False

## Neoliberal Museum Analysis

It's not entirely clear what the income column represents in the IMLS data. So perhaps this isn't a good idea.Since we have the income for the museum and the total size (bytes) retrieved from their website we can calculate a ratio of the gigabytes per dollar on the museum website. Then we can sort it so that the institutions that are making the most amount of data relative to their income: *gigabytes per dollar*.

In [4]:
imls = imls.assign(size_gb = imls['Size'] / 1024 / 1024 / 1024)
imls = imls.assign(size_ratio = imls["size_gb"]/ imls["Income"])
imls.sort_values(by="size_ratio", ascending=False)

Name  \
507                         VALLEY INSTITUTE VISUAL ART   
5516                            COLLEGE OF EASTERN UTAH   
3034               SHELBY COUNTY HISTORICAL SOCIETY INC   
143                            TEMPE HISTORICAL SOCIETY   
337            ALLIED GARDENS PARK & RECREATION COUNCIL   
1869     ALLEN COUNTY COURTHOUSE PRESERVATION TRUST INC   
2841                  MUSKEGON MUSEUM OF ART FOUNDATION   
3526    FOUNDATION FOR THE CHAPEL OF SACRED MIRRORS LTD   
1568               MUSEUM OF CLASSIC CHICAGO TELEVISION   
5266                         MUSEUM OF COMPUTER CULTURE   
2008                  SHAWNEE INDIAN MISSION FOUNDATION   
3993                         BUNGERS SURFING FOUNDATION   
3375                FRIENDS OF ABRAHAM STAATS HOUSE INC   
2359                TRUSTEES OF THE WALTERS ART GALLERY   
2486                   SALISBURY HISTORICAL SOCIETY INC   
2025                    RILEY COUNTY HISTORICAL SOCIETY   
474             FRIENDS OF COLUMBIA STATE HISTORIC PARK   
6035                OSHKOSH PUBLIC MUSEUM AUXILIARY INC   
431               THE DELTA SCIENCE CENTER AT BIG BREAK   
4343                            EARLY TELEVISION MUSEUM   
3639          LOWER HUDSON VALLEY CHALLENGER CENTER INC   
5909                          ART WORKS AROUND TOWN INC   
3946        ALUMNI SOCIETY OF THE SCHOOL OF VISUAL ARTS   
4539                OREGON JOHN DEERE MUSEUM FOUNDATION   
4515                  STILLWATER HIGH SCHOOL MUSEUM INC   
1840              INDIANA JEWISH HISTORICAL SOCIETY INC   
3291                    FRIENDS OF THE LIBBY MUSEUM INC   
693                            FOLK MUSIC CENTER MUSEUM   
4570              UNIVERSITY OF PORTLAND BOTANIC GARDEN   
1982                       OLD BRICK EPISCOPAL LUTHERAN   
...                                                 ...   
4557                       USS RANGER MUSEUM FOUNDATION   
4556              OREGON COAST MILITARY HERITAGE MUSEUM   
2328                                   BLACK IS A COLOR   
2330  AFRICAN-AMERICAN SCHOOLHOUSE MUSEUM & HERITAGE...   
4553              FRIENDS OF THE HISTORIC GRAND THEATER   
4552                                 THE GEEZER GALLERY   
2331                       ROCKVILLE SCIENCE CENTER INC   
2332               SOUTHERN MARYLAND CAROUSEL GROUP INC   
4549                      TOASTER MUSEUM FOUNDATION INC   
4548                             FRIENDS OF@TWO BRIDGES   
2333                       CECIL COUNTY FARM MUSEUM INC   
2334                      DOLEMAN BLACK HERITAGE MUSEUM   
4545                  YAMHILL COUNTY HISTORICAL SOCIETY   
4542                           CLATSOP CRUISE HOSTS INC   
2372                                   CASA OF MARYLAND   
2338              WASHINGTON COUNTY MUSEUM OF FINE ARTS   
4540                          OREGON VOLLEYBALL ACADEMY   
2344                     BALTIMORE STREETCAR MUSEUM INC   
2345             ST MARYS COUNTY HISTORICAL SOCIETY INC   
2347       PEERLESS ROCKVILLE HISTORIC PRESERVATION LTD   
2352          THE GARRETT COUNTY HISTORICAL SOCIETY INC   
4534                          MCLOUD HISTORICAL SOCIETY   
2355        WESTMINSTER PRESERVATION TRUST INCORPORATED   
2356                                HISTORIC TAKOMA INC   
2364              HISTORIC ST MARYS CITY FOUNDATION INC   
4530                    NO MANS LAND HISTORICAL SOCIETY   
2365                  BALTIMORE PUBLIC WORKS MUSEUM INC   
2369                   THE AMERICAN FLAG FOUNDATION INC   
2370  PARKS AND PEOPLE FOUNDATION FOR BALTIMORE RECR...   
0                                            ARTSREVIVE   

                                       URL     Income   Pages         Size  \
507                    www.vivagallery.org       22.0    2153    123098457   
5516                           www.ceu.edu      336.0   22676   1589230787   
3034                   shelby.mogenweb.org     7156.0   10980  22380537929   
143                          www.tempe.gov     6053.0  119784   5044019549   
33

This is a bit weird because there are institutions that have an income of \$22.00. So let's look at museums that have an income of more than \$100,000.00

In [5]:
large_imls = imls[imls['Income'] > 100000]
large_imls.sort_values(by='size_ratio', ascending=False)

Name  \
337            ALLIED GARDENS PARK & RECREATION COUNCIL   
2025                    RILEY COUNTY HISTORICAL SOCIETY   
3946        ALUMNI SOCIETY OF THE SCHOOL OF VISUAL ARTS   
2374                JEFFERSON PATTERSON PARK AND MUSEUM   
5211                  TEXAS ARCHIVE OF THE MOVING IMAGE   
1660                         ELMHURST HISTORICAL MUSEUM   
4173                       TOBACCO FARM LIFE MUSEUM INC   
4594                 GILBERT HOUSE CHILDRENS MUSEUM INC   
710                    HERITAGE MUSEUM OF ORANGE COUNTY   
1707                      ELGIN AREA HISTORICAL SOCIETY   
1214                AMELIA ISLAND MUSEUM OF HISTORY INC   
2725                   YPSILANTI HISTORICAL SOCIETY INC   
3295           MOUNT WASHINGTON VALLEY CHILDRENS MUSEUM   
549                          ANTIOCH HISTORICAL SOCIETY   
1692                            ILLINOIS RAILWAY MUSEUM   
2076                    KANSAS STATE HISTORICAL SOCIETY   
2825                        RAVEN HILL DISCOVERY CENTER   
645               CALIFORNIA MILITARY MUSEUM FOUNDATION   
2611                               HISTORIC NORTHAMPTON   
3867                          FENTON HISTORICAL SOCIETY   
3882  MALCOLM X DR BETTY SHABAZZ MEMORIAL EDUCATIONA...   
2031                CHEROKEE STRIP LAND RUSH MUSEUM INC   
6064                                   THE MURIE CENTER   
2924                   WASECA COUNTY HISTORICAL SOCIETY   
5735                          GEORGE C. MARSHALL MUSEUM   
4879                  KEMERER MUSEUM OF DECORATIVE ARTS   
2143                                  BRENNAN HOUSE INC   
4937                          ALLISON-ANTRIM MUSEUM INC   
1361                     DEKALB HISTORICIAL SOCIETY INC   
806   FRIENDS OF THE COLORADO SPRINGS PIONEERS MUSEU...   
...                                                 ...   
4809                                 AMBLER THEATER INC   
2427  NATIONAL CAPITAL HISTORICAL MUSEUM OF TRANSPOR...   
4806                        NEW HOPE HISTORICAL SOCIETY   
4804   SEMINARY RIDGE HISTORIC PRESERVAT ION FOUNDATION   
2432                          COASTAL HERITAGE ALLIANCE   
2433  ANNAPOLIS LONDON TOWN & SOUTH COUNTY HERITAGE ...   
2458                          HOUSE OF THE SEVEN GABLES   
4793                       PITTSBURGH PARKS CONSERVANCY   
4792                  PHILADELPHIA MURAL ARTS ADVOCATES   
2472                      FALL RIVER HISTORICAL SOCIETY   
2394                 LOCUST POINT CIVIC ASSOCIATION INC   
2384                           PROJECT LIBERTY SHIP INC   
2338              WASHINGTON COUNTY MUSEUM OF FINE ARTS   
2372                                   CASA OF MARYLAND   
4857                   SUN INN PRESERVATION ASSOCIATION   
4854                       BUCKS COUNTY AUDUBON SOCIETY   
4853  FOCUS ON RENEWAL STO-ROX NEIGHBORHOOD CORPORATION   
2344                     BALTIMORE STREETCAR MUSEUM INC   
2347       PEERLESS ROCKVILLE HISTORIC PRESERVATION LTD   
2352          THE GARRETT COUNTY HISTORICAL SOCIETY INC   
2355        WESTMINSTER PRESERVATION TRUST INCORPORATED   
2356                                HISTORIC TAKOMA INC   
2364              HISTORIC ST MARYS CITY FOUNDATION INC   
4839                   CHESTNUT HILL HISTORICAL SOCIETY   
4837  HISTORIC PRESERVATION TRUST OF BERKS COUNTY PE...   
4836    DELAWARE COUNTY HISTORICAL SOCIETY PENNSYLVANIA   
4835                    HANOVER AREA HISTORICAL SOCIETY   
4833             JEWISH FEDERATION OF THE LEHIGH VALLEY   
2370  PARKS AND PEOPLE FOUNDATION FOR BALTIMORE RECR...   
0                                            ARTSREVIVE   

                                     URL     Income   Pages         Size  \
337                     www.sandiego.gov   113766.0   53357  74895087335   
2025                        www.kshs.org   113926.0   60364  14781730183   
3946                         www.sva.edu   393953.0   49712  38945037160   
2374                      www.jefpat.org   162410.0   46033   8462341995   
5211        

We can also calculate a similar ratio for the number of pages per dollar, and get the top results.

In [6]:
imls = imls.assign(pages_ratio = imls['Pages'] / imls['Income'])
imls.sort_values(by='pages_ratio', ascending=False)

Name  \
507                         VALLEY INSTITUTE VISUAL ART   
5516                            COLLEGE OF EASTERN UTAH   
1568               MUSEUM OF CLASSIC CHICAGO TELEVISION   
143                            TEMPE HISTORICAL SOCIETY   
3375                FRIENDS OF ABRAHAM STAATS HOUSE INC   
5909                          ART WORKS AROUND TOWN INC   
386                  TREASURE ISLAND MUSEUM ASSOCIATION   
6035                OSHKOSH PUBLIC MUSEUM AUXILIARY INC   
3993                         BUNGERS SURFING FOUNDATION   
1280                 CIVIC MEDIA CENTER AND LIBRARY INC   
4173                       TOBACCO FARM LIFE MUSEUM INC   
861   SANGRE DECRISTO ARTS & CONFERENCE CENTER ENDOW...   
547                    RIO VISTA MUSEUM ASSOCIATION INC   
877                PUEBLO ZOOLOGICAL SOCIETY FOUNDATION   
3882  MALCOLM X DR BETTY SHABAZZ MEMORIAL EDUCATIONA...   
5266                         MUSEUM OF COMPUTER CULTURE   
3994                WALT WHITMAN BIRTHPLACE ASSOCIATION   
4570              UNIVERSITY OF PORTLAND BOTANIC GARDEN   
2143                                  BRENNAN HOUSE INC   
1214                AMELIA ISLAND MUSEUM OF HISTORY INC   
1884                    WORLD OF MOTORCYCLES MUSEUM INC   
2841                  MUSKEGON MUSEUM OF ART FOUNDATION   
1869     ALLEN COUNTY COURTHOUSE PRESERVATION TRUST INC   
1982                       OLD BRICK EPISCOPAL LUTHERAN   
3034               SHELBY COUNTY HISTORICAL SOCIETY INC   
4744                   SCHUYLKILL CANAL ASSOCIATION INC   
6064                                   THE MURIE CENTER   
3295           MOUNT WASHINGTON VALLEY CHILDRENS MUSEUM   
4594                 GILBERT HOUSE CHILDRENS MUSEUM INC   
3945                    NORTH CASTLE HISTORICAL SOCIETY   
...                                                 ...   
2258                     CASTINE HISTORICAL SOCIETY INC   
2257                           BOOTHBAY RAILWAY VILLAGE   
2256          WENDELL GILLEY MUSEUM OF BIRD CARVING THE   
4490                      DURANT HISTORICAL SOCIETY INC   
2254                     OLD BERWICK HISTORICAL SOCIETY   
4464                     WESTERVILLE HISTORICAL SOCIETY   
4493               ROGERS COUNTY HISTORICAL SOCIETY INC   
4494                 EDMOND HISTORIC PRESERVATION TRUST   
2253                       NOBLEBORO HISTORICAL SOCIETY   
2251                        MOOSEHEAD MARINE MUSEUM INC   
2249                        YARMOUTH HISTORICAL SOCIETY   
4499                       NATIONAL BIPLANE ASSOCIATION   
2269     NATURAL RESOURCE EDUCATION CENTER AT MOOSEHEAD   
2270                          MCLAUGHLIN FOUNDATION INC   
4481                                UNIVERSITY OF TULSA   
4480         JEWISH FEDERATION OF GREATER OKLAHOMA CITY   
4479                        OKLAHOMA CITY MUSEUM OF ART   
4478                              ARTIST INCUBATION INC   
2272                                   OPERA HOUSE ARTS   
4476       MCCURTAIN COUNTY VETRANS MEMORIAL MUSEUM INC   
2275                                 PINELAND FARMS INC   
4473  MOST WORSHIPFUL GRAND LODGE AF & AM OF STATE O...   
4472                            MUSEUM OF THE BIBLE INC   
2276                             WATERVILLE MAIN STREET   
2277                      WATERVILLE HISTORICAL SOCIETY   
2278                     UNIVERSITY OF MAINE FOUNDATION   
2282                  MAINE FOREST & LOGGING MUSEUM INC   
4466               ST JOSEPH-SALEM HERITAGE SOCIETY INC   
2284                  SOUTH PORTLAND HISTORICAL SOCIETY   
0                                            ARTSREVIVE   

                                          URL       Income   Pages  \
507                       www.vivagallery.org         22.0    2153   
5516                              www.ceu.edu        336.0   22676   
1568                     www.fuzzymemories.tv      12385.0  813857   
143                             www.tempe.gov       6053.0  119784   
3375                          staatshouse.

## Integrity

Since we have the total pages requested from a domain, and the total number of 404s at that domain we can calcuate a new column *missing_ratio* . The lower this ratio, the better the link integrity in the website. We can then sort our table in ascending order and see which museum websites have the best link integrity.

In [7]:
imls = imls.assign(missing_ratio = imls["404"] / imls["Pages"])
imls.sort_values(by='missing_ratio')

Name  \
2566              NEW BEDFORD MUSEUM AND ART CENTER INC   
3884                             STEEL PLANT MUSEUM INC   
3882  MALCOLM X DR BETTY SHABAZZ MEMORIAL EDUCATIONA...   
848           SANGRE DE CRISTO ARTS & CONFERENCE CENTER   
1522   NEZ PERCE COUNTY HISTORICAL SOCIETY INCORPORATED   
850                      CHILDRENS MUSEUM OF DENVER INC   
852                           PUEBLO ZOOLOGICAL SOCIETY   
854                         FOUR MILE HISTORIC PARK INC   
3880                             MUSEUM OF BIBLICAL ART   
3867                          FENTON HISTORICAL SOCIETY   
5415                     SAN ANGELO MUSEUM OF FINE ARTS   
5422              HISTORIC AVIATION MEMORIAL MUSEUM INC   
2922                   LAKE SUPERIOR ZOOLOGICAL SOCIETY   
3864        GLENN H CURTISS MUSEUM OF LOCAL HISTORY INC   
3861                          AURORA HISTORICAL SOCIETY   
3859                          JAMESTOWN AUDUBON SOCIETY   
1519                   POCATELLO ZOOLOGICAL SOCIETY INC   
1515               SIERRA SILVER MINE TOUR INCORPORATED   
2924                   WASECA COUNTY HISTORICAL SOCIETY   
5561                       BARRE HISTORICAL SOCIETY INC   
872                      CASTLE ROCK HISTORICAL SOCIETY   
2941                SHERBURNE COUNTY HISTORICAL SOCIETY   
876          CRESTED BUTTE MOUNTAIN HERITAGE MUSEUM INC   
877                PUEBLO ZOOLOGICAL SOCIETY FOUNDATION   
3887  HENDRICK HUDSON HIGH SCHOOL ATHLETIC HALL OF FAME   
879            KIRKLAND MUSEUM OF FINE & DECORATIVE ART   
3892  DISCOVERY CENTER OF THE SOUTHERNTIER FOUNDATIO...   
841                      BOULDER HISTORICAL SOCIETY INC   
810                GREELEY FREIGHT STATION MUSEUM GUILD   
3940       HISTORICAL SOCIETY OF GREATER PORT JEFFERSON   
...                                                 ...   
5985                 WEST BEND FRIENDS OF SCULPTURE INC   
5992                       MILITARY VETERANS MUSEUM INC   
5993             PORT WASHINGTON HISTORICAL SOCIETY INC   
5995                               FRIENDS OF THE DOMES   
5998  FRANK LLOYD WRIGHT WISCONSIN HERITAGE TOURISM ...   
6000            SHEBOYGAN COUNTY CHAMBER FOUNDATION INC   
6001                           FOND DU LAC ARTS COUNCIL   
6005                        TEN CHIMNEYS FOUNDATION INC   
6006           CAMBRIDGE HISTORIC SCHOOL FOUNDATION INC   
6007          WISCONSIN STATE FIREFIGHTERS MEMORIAL INC   
6008                       LITTLE WHITE SCHOOLHOUSE INC   
6013                      SWISS CENTER OF NORTH AMERICA   
6021          FOND DU LAC COUNTY HISTORICAL SOCIETY INC   
6029                     POLK COUNTY HISTORICAL SOCIETY   
6030                                RACINE ARTS COUNCIL   
6031  DEPERE HISTORICAL SOCIETY WHITE PILLARS ENDOW ...   
6034                   GREENDALE HISTORICAL SOCIETY INC   
6037        MUKWONAGO HISTORICAL AND MUSEUM SOCIETY INC   
6041                          ROBERT W MONK GARDENS INC   
6042          STEAM LOCOMOTIVE HERITAGE ASSOCIATION INC   
6045                                       SCIENCE ZONE   
6046                             MUSEUM OF THE OLD WEST   
6047                 BRADFORD BRINTON MEMORIAL 01-26150   
6048           NEW MUSEUM AT THE BRADFORD BRINTON RANCH   
6049                          CHEYENNE DEPOT MUSEUM INC   
6051                 SHERIDAN COUNTY LIBRARY FOUNDATION   
6053                FORT LARAMIE HISTORICAL ASSOCIATION   
6055             CHEYENNE FRONTIER DAYS OLD WEST MUSEUM   
6058                      GREEN RIVER VALLEY MUSEUM INC   
6063                            LANDER CHILDRENS MUSEUM   

                                       URL     Income   Pages        Size  \
2566           www.newbedfordartmuseum.org   328130.0       3        1989   
3884                www.steelpltmuseum.org    42806.0       3        1540   
3882              www.theshabazzcenter.net   312303.0  867395  3829017053   
848                 sangredecristoarts.org  1494933.0       3        1523   
1522   

The first two rows aren't interesting because the websites only had 3 pages. But checkout the [Malcolm X and Dr Betty Shabazz Memorial and Education Center](http://www.theshabazzcenter.net/) which has 867,395 pages and **ZERO** 404 errors.

We can limit our results to websites with more than 100,000 pages.

In [8]:
large_imls = imls[imls['Pages'] > 100000]
large_imls.sort_values(by='missing_ratio')

Name  \
2484                    JOHN WOODMAN HIGGINS ARMORY INC   
3882  MALCOLM X DR BETTY SHABAZZ MEMORIAL EDUCATIONA...   
3867                          FENTON HISTORICAL SOCIETY   
3686                            ANTHOLOGY FILM ARCHIVES   
5946                    WISCONSIN HISTORICAL FOUNDATION   
4937                          ALLISON-ANTRIM MUSEUM INC   
1568               MUSEUM OF CLASSIC CHICAGO TELEVISION   
1402  MUSEUM OF AVIATION AT ROBINS AIR FORCE BASE GE...   
2143                                  BRENNAN HOUSE INC   
2441                 PROPRIETORS OF THE SALEM ATHENAEUM   
3212        GREAT PLATTE RIVER ROAD MEMORIAL FOUNDATION   
4173                       TOBACCO FARM LIFE MUSEUM INC   
2067                         KANSAS AVIATION MUSEUM INC   
4594                 GILBERT HOUSE CHILDRENS MUSEUM INC   
148                              ARIZONA SCIENCE CENTER   
2615                               FITCHBURG ART MUSEUM   
1692                            ILLINOIS RAILWAY MUSEUM   
1005                                 AMISTAD FOUNDATION   
2399                     BALTIMORE CHILDRENS MUSEUM INC   
2907                  OLMSTED COUNTY HISTORICAL SOCIETY   
338                                   USS MIDWAY MUSEUM   
5909                          ART WORKS AROUND TOWN INC   
4679                         PHILADELPHIA MUSEUM OF ART   
3955              HANFORD MILLS MUSEUM AT EAST MEREDITH   
1270                EXPLORATIONS V CHILDRENS MUSEUM INC   
172                         CHILDRENS MUSEUM OF PHOENIX   
4076                 TRUSTEES OF THE SCHIELE MUSEUM INC   
806   FRIENDS OF THE COLORADO SPRINGS PIONEERS MUSEU...   
4377                                   AKRON ART MUSEUM   
3031     FRIENDS OF THE ZOO OF SPRINGFIELD MISSOURI INC   
...                                                 ...   
4384                       NATIONAL FOOTBALL MUSEUM INC   
3295           MOUNT WASHINGTON VALLEY CHILDRENS MUSEUM   
5855                           BURKE MUSEUM ASSOCIATION   
5274                        MUSEUM OF FINE ARTS HOUSTON   
313                 CHARLES W BOWERS MUSEUM CORPORATION   
2530                     NEW ENGLAND MUSEUM ASSOCIATION   
2904                       MINNESOTA HISTORICAL SOCIETY   
3806                             GENESEE COUNTRY MUSEUM   
5155                            UNIVERSITY OF THE SOUTH   
3858                          BUFFALO FINE ARTS ACADEMY   
3561                          AUDUBON SOCIETY, NATIONAL   
1747      ILLINOIS STATE MUSEUM SOCIETY MUSEUM BUILDING   
2900                                  WALKER ART CENTER   
1932                                     LUTHER COLLEGE   
3522                            NEW YORK TRANSIT MUSEUM   
2470                   MASSACHUSETTS HISTORICAL SOCIETY   
1400                 TUBMAN AFRICAN AMERICAN MUSEUM INC   
143                            TEMPE HISTORICAL SOCIETY   
2450                       AMERICAN ANTIQUARIAN SOCIETY   
3965                INTERNATIONAL CENTER OF PHOTOGRAPHY   
5790                        TACOMA ART MUSEUM TACOMA WA   
1176                              MUSEUM OF SCIENCE INC   
1196              DUNEDIN FINE ARTS AND CULTURAL CENTER   
2055                              EXPLORATION PLACE INC   
3970                                SEWARD HOUSE MUSEUM   
3662                                NEW YORK UNIVERSITY   
1929                         CEDAR RAPIDS MUSEUM OF ART   
1869     ALLEN COUNTY COURTHOUSE PRESERVATION TRUST INC   
6064                                   THE MURIE CENTER   
660                                 J. PAUL GETTY TRUST   

                                   URL        Income   Pages         Size  \
2484                   www.higgins.org  1.392623e+06  949328   1640204637   
3882          www.theshabazzcenter.net  3.123030e+05  867395   3829017053   
3867       www.fentonhistorycenter.org  2.003000e+05  175516   2523574453   
3686     www.anthologyfilmarchives.org  7.898710e+05  868477   4634327128   
5946   

Hopefully this provides a little glimmer of an example of how web archives could be used to study Museum websites.